migrate.py

```diff
@@ -363,45 +372,68 @@ def convert_comment(comment, options):
     }
 
 
-def format_issue_body(issue, options):
-    content = convert_changesets(issue['content'])
-    content = convert_creole_braces(content)
-    content = convert_links(content, options)
-    content = convert_users(content, options)
-    return """*Originally reported by* **{reporter}**
+SEP = "-" * 40
+
+ISSUE_HEADER = """*Originally reported by* **{reporter}**
 
 {sep}
 
-{content}
+"""
+
+ISSUE_CONTENT = """{content}
 
 {sep}
 - Bitbucket: https://bitbucket.org/{repo}/issue/{id}
-""".format(
+"""
+
+COMMENT_HEADER = """*Original comment by* **{author}**
+
+{sep}
+
+"""
+
+COMMENT_CONTENT = """{content}
+"""
+
+def format_issue_body(issue, options):
+    content = convert_changesets(issue['content'])
+    content = convert_creole_braces(content)
+    content = convert_links(content, options)
+    content = convert_users(content, options)
+    reporter = issue.get('reported_by', None)
```

Comment:
No need for `None`, it's default. I don't think making it explicit here adds much readability...

Reply:
That was in the original code, but I can fix it :)

Reply:
Oops, I guess my pythonic-ness has improved over the past few years. 😃 Thanks for fixing.